desc/compute/_curve.py

```diff
@@ -146,6 +146,50 @@ def _Z_Curve(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="center",
+    label="\\langle\\mathbf{x}\\rangle",
+    units="m",
+    units_long="meters",
+    description="Centroid of the curve",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="s",
+    data=["x"],
+    parameterization="desc.geometry.core.Curve",
+)
+def _center_Curve(params, transforms, profiles, data, **kwargs):
+    data["center"] = jnp.mean(data["x"], axis=0) * jnp.ones_like(data["x"])
```

Comment:
this will be weird if the grid only has 1 point. It might be better to make specialized versions for the different classes that get the n=0 fourier mode or average of the spline knots etc.

Reply:
must be in xyz for this to work too